aiida_optimade/routers/info.py

```diff
@@ -30,6 +30,10 @@ def get_info(request: Request):
     parse_result = urllib.parse.urlparse(str(request.url))
     base_url = get_base_url(parse_result)
 
+    root_path_str = ""
+    if CONFIG.root_path:
+        root_path_str = f"/{CONFIG.root_path.strip('/')}"
```

Comment:
You can use https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urljoin to generate the url with `/` properly striped.